pkgs/development/libraries/vtk/generic.nix

```diff
@@ -48,6 +61,8 @@ in stdenv.mkDerivation {
       GLUT
     ] ++ optionals enablePython [
       python
+    ] ++ optionals enableVtkDicom [
+      (gdcm.override { enableVTK = false; })
```

Comment:
maybe we should consider lifting this to e.g. all-packages.nix say by defining vtkWithGdcm or similar, but I think it's fine?

Reply:
It's vice versa, `gdcmWithGtk`. I have no preference, so can move it.